core/templates/dev/head/forms/formBuilder.js

```diff
@@ -935,7 +935,10 @@ oppia.directive('textAngularRte', [
           };
 
           $scope.stripFormatting = function(html) {
-            return $filter('sanitizeHtmlForRte')(html);
+            // Strip out anything between and including <>,
+            // unless it contains the RTE tags.
+            return html ? String(html)
+              .replace(/(?!.*(\"oppia-noninteractive-.))<[^>]+>/gm, '') : '';
```

Comment:
Only one suggestion -- instead of accepting anything with a oppia-noninteractive- prefix, can we accept only the six specific tags from getRichTextComponents() and reject anything else? Maybe declare a whitelistedTags variable, populate that variable with those specific tags, and exclude anything but those. This is stricter and will also allow us to add/remove tags from that whitelist in the future.